src/index.ts

```diff
@@ -14,6 +14,10 @@ import { inspect } from "node:util";
 // twice.
 const ENV_DAEMON_DIR = "MAGIC_NIX_CACHE_DAEMONDIR";
 
+const FACT_AUTHENTICATED_ENV = "authenticated_env";
```

Comment:
what does this mean exactly? Is there a more descriptive name?

Reply:
I'm not sure, all I've done here is `const`ify the key. Open to suggestion.

Reply:
I'm still not sure why we need this but I've settled on `required_env_vars_present` for now.